docs/pages/eas/workflows/syntax.mdx

```diff
@@ -384,6 +384,33 @@ jobs:
       prod: boolean # optional
 ```
 
+#### `fingerprint`
+
+```yaml
+jobs:
+  my_job:
+    # @info #
+    type: fingerprint
+    # @end #
+```
+
+Calculates fingerprint of iOS and Android builds.
```

Comment:
```suggestion
Calculates fingerprint of Android and iOS builds.
```

Vale fix.

Reply:
Interesting to see that Vale mentions the wrong line 😅 Not sure what's going on but will have a look later if I see another one.